components/Cards/cards.css

```diff
@@ -0,0 +1,116 @@
+/* global css sheet for fonts and icons */
+@import "/css/main.css";
+
+/* main card component */
+.card {
+    height: 14rem;
+    width: 40rem;
+    border: 2px solid var(--universal-border-color);
+    margin: 0.5rem;
+    /* overflow: auto; */
+}
+
+/* image HTML5 tag */
+img {
+    height: 13rem;
+    width: 15rem;
+    border: 2px solid var(--universal-border-color);
+    margin: 0.3rem;
+}
+
+/* card content class */
+.card-content {
+    display: flex;
+    flex-direction: column;
+    margin-top: 0.3rem;
+    margin-left: 0.2rem;
+}
+
+/* all important classes groped togther */
+.card-with-badge,
+.btn-icons,
+.card-dismiss,
+.card-with-text-overlay,
+.card-with-text-only,
+.card-horizontal {
+    display: flex;
+}
+
+/* HTML5 btton element */
+button {
+    height: 2rem;
+    margin: 0.3rem 0.3rem 0 0;
+    display: flex;
+    align-items: center;
+    border-radius: 0.3rem;
+    cursor: pointer;
+}
+
+/* button class for cart and wishlist */
+.btn-add-to-cart,
+.btn-add-to-wishlist {
+    background-color: var(--shopping-card-button-bgcolor);
+    color: var(--background-universal-color);
+}
+
+/* small text class */
+.text-light {
+    font-size: 0.8rem;
+}
+
+/* badge class */
+.badge-verify {
+    position: absolute;
+    left: 14rem;
+    top: 3.5rem;
+    color: var(--success-alert-icon-color);
+    cursor: pointer;
+}
+
+.badge-close {
+    position: relative;
+    left: 38rem;
+    top: 3rem;
+    color: var(--danger-alert-icon-color);
+    cursor: pointer;
+}
+
+/* card textoverlay class */
+.card-with-text-overlay {
+    opacity: 0.5;
+}
+
+.text-overlay-over-card::after {
+    height: 9rem;
+    width: 15rem;
+    display: flex;
+    align-items: center;
+    content: 'Out of Stock';
+    background-image: linear-gradient(to right, red, rgb(214, 214, 34));
+    font-weight: bold;
+    font-size: 2.5rem;
+    position: relative;
+    top: -10rem;
+    right: 5rem;
+    opacity: 0.8;
+}
+
+/* other card classes */
+.card-vertical,
+.card-with-shadow {
+    display: flex;
+    flex-direction: column;
+    margin: 0.5rem;
+}
+
+/* vertical card class */
+.vcard {
+    height: 26rem;
+    width: 16rem;
+    border: 2px solid var(--universal-border-color);
+}
```

Comment:
Same remove the width property because you don't want your card to be of fixed width